docs/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Build:
+# docker build -t ace3mod/jekyll .
+#
+# Run:
+# docker run -p 4000:4000 -it ace3mod/jekyll
+
+FROM starefossen/github-pages:latest
+MAINTAINER bux
+
+VOLUME "/usr/src/app"
```

Comment:
Should we also copy the contents of `.` to `/usr/src/app` ? Means you won't have too mount in a volume always, but still can.